test/tests.json

```diff
@@ -606,5 +606,23 @@
       "compact"
     ],
     "workdir": "rs/bin"
+  },
+  {
+    "name": "secure",
+    "client": {
+      "command": [
+        "test_secure.bash"
+      ]
+    },
+    "transports": [
+      "buffered"
+    ],
+    "sockets": [
+      "ip-ssl"
+    ],
+    "protocols": [
+      "binary"
+    ],
+    "workdir": "secure"
```

Comment:
I do not see where crossfeature is executed during Travis CI.  Which build job runs it?